### PR TITLE
status: 2023q2: OpenStack: markup et cetera

### PR DESCRIPTION
**Pull request by grahamperrin**
Markup.

Tense.

Other changes, minor.

### CODE REVIEW

**Comment by grahamperrin** [comment]
@lwhsu

File: website/content/en/status/report-2023-04-2023-06/openstack.adoc [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 
-In the next quarter, we will continue working on the console proxy services to make the overall workflow more fluent. 
+In the next quarter, we will continue working on the console proxy services to make the overall workflow more fluent.
 
-The step-by-step documents for constructing a POC site can also be found link:https://github.com/openstack-on-freebsd/docs[in the `docs` repository]. 
-The patched version of each OpenStack component is under the same GitHub organization. 
+The step-by-step documents for constructing a POC site can also be found link:https://github.com/openstack-on-freebsd/docs[in the `docs` repository].
+The patched version of each OpenStack component is under the same GitHub organization.
 
 People interested in helping with the project can first help check the documentation by following the installation guide.
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-04-2023-06/openstack.adoc:27]: I think there is an extra space at the end of this line too.

**Review by lsalvadore — APPROVED** [review]
Approved, conditionnaly to @starbops 's (original report submitter) approval for the subjective changes.

File: website/content/en/status/report-2023-04-2023-06/openstack.adoc (hunk 1 above)
Comment (grahamperrin) [reply website/content/en/status/report-2023-04-2023-06/openstack.adoc]: Resolved by [`5a643e2` (#209)](https://github.com/freebsd/freebsd-doc/pull/209/commits/5a643e25274e89c87c2eed9f7a512b8154ca2bc6)

**Comment by lsalvadore** [comment]
Merged, thanks.

**Pull request closed without merging**